src/components/Main/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Button, ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap'
 import './index.css'
-
+import tokenSales from '../TokenSale/tokenSales.js'
```

Comment:
No need of using extension .js, if you run `yarn lint:js`, the linter will hint you as well.

Reply:
gotcha np